content/guidance/inventory-code.md

```diff
@@ -13,4 +13,172 @@ sidenav: true
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <table class="usa-table">
+    <thead>
+      <tr>
+        <th scope="col">Field Name</th>
+        <th scope="col">Contact</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Data Type</th>
+        <td>Object</td>
+      </tr>
+      <tr>
+        <th scope="row">Definition</th>
+        <td>Point of contact information for the release.</td>
+      </tr>
+      <tr>
+        <th scope="row">Required</th>
+        <td>Always</td>
+      </tr>
+      <tr>
+        <th scope="row"><span style="white-space: nowrap;">AI R&amp;D Guidance</span></th>
+        <td>
+          Be sure to identify and include domain experts and their contact information who can discuss the model with interested AI researchers. If not the same as the domain expert, Agencies shall also identify an expert and their contact information who can discuss restrictions or controls on the model.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <table class="usa-table">
+    <thead>
+      <tr>
+        <th scope="col">Field Name</th>
+        <th scope="col">exemptionText</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Data Type</th>
+        <td>String or Null</td>
+      </tr>
+      <tr>
+        <th scope="row">Definition</th>
+        <td>
+          If an exemption is listed in the <code>usageType</code> field, this field should include a one or two sentence justification for the exemption used.
+        </td>
+      </tr>
+      <tr>
+        <th scope="row">Required</th>
+        <td>No</td>
+      </tr>
+      <tr>
+        <th scope="row"><span style="white-space: nowrap;">AI R&amp;D Guidance</span></th>
+        <td>
+          Agencies shall describe how researchers may be able to access
+          <code>governmentWideReuse</code> or exempt data.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+  <table class="usa-table">
+    <thead>
+      <tr>
+        <th scope="col">Field Name</th>
+        <th scope="col">relatedCode</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row">Data Type</th>
+        <td>Array</td>
+      </tr>
+      <tr>
+        <th scope="row">Definition</th>
+        <td>
+          An array of affiliated government repositories that may be a part of the same project. For example, <code>relatedCode</code> for <code>code-gov-front-end</code> would include <code>code-gov-api</code> and <code>code-gov-client</code>.
+        </td>
+      </tr>
+      <tr>
+        <th scope="row">Required</th>
+        <td>No</td>
+      </tr>
+      <tr>
+        <th scope="row"><span style="white-space: nowrap;">AI R&amp;D Guidance</span></th>
+        <td>Agencies shall describe related models and code.</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+</div><div id="how-to-inventory-b"><div><h2 id="examplecodejsonfile">Example code.json file</h2>
```

Comment:
This is not aligned with the rest of the content in the page so I would move this to under the table tag in line 172